tenable/io/v3/was/__init__.py

```diff
@@ -17,9 +17,11 @@
     :glob:
 
     folders
+    plugins
 '''
 from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
 from tenable.io.v3.was.folders.api import FoldersAPI
+from tenable.io.v3.was.plugins.api import PluginsAPI
```

Comment:
Can we make a generic base class here for both VM and WAS?

Reply:
@tomeara Could you please let us know the purpose for keeping the generic class for WAS and VM? We have not kept a generic class for the following reasons:

- The complete documentation is generated based on sphinx, and it leverages the comments from the modules directly.
- So, for example, if the details method will have different usage for was: tio.v3.was.plugins.details(1) and VM: tio.v3.vm.plugins.details(1), it would be easier for users to have that documented here. Otherwise, it won't be visible in this module, but the user will have to refer to the parent class.
- Also, for plugins, we need to do minimum processing, and we are directly making the API call. So, the code is also not duplicated. 

Please let us know if you still think we should go for keeping generic class.

Reply:
I suppose that's fine